tests/Fixtures/app/object-manager.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+// See https://github.com/phpstan/phpstan-doctrine#configuration
+
+$kernel = new AppKernel($_SERVER['APP_ENV'] ?? 'test', (bool) ($_SERVER['APP_DEBUG'] ?? false));
+$kernel->boot();
+
+return $kernel->getContainer()->get('doctrine')->getManager();
```

Comment:
As discussed with @laryjulien, I think we need to do something more complex here:

1. create a new `PHPSTAN` Symfony environment loading both the ORM and the ODM (currently we can only load one of them)
2. create an implementation of the `ObjectManagerInterface` here that will use composition to dispatch between the ORM and the ODM one (this can be an anonymous class)

This should allow the PHPStan Doctrine plugin to work properly with entities and documents.